index.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express'
+
+const app = express();
+const PORT = process.env.PORT ?? 3000;
+
+app.use(express.json());
+
+app.get('/api/greetings', (request, response) => {
+  const name = request.query.name;
```

Comment:
We can destructure prop from query object:
```suggestion
  const { name } = request.query;
```